### ISSUE TITLE
Failure to load, libc++_shared.so not found

### ISSUE THREAD

**Issue by westnordost** [tangrams/tangram-es#1426]
A crash was reported for my app. So far, it happened for the following devices:

- ALCATEL ONE TOUCH 6040A (DIABLOX), Android 4.2, native platform: armeabi-v7a
- IliumPAD, Android 4.2 (no further info available)

```
java.lang.UnsatisfiedLinkError: Cannot load library: soinfo_link_image(linker.cpp:1635): could not load library "libc++_shared.so" needed by "libtangram.so"; caused by load_library(linker.cpp:745): library "libc++_shared.so" not found
	at java.lang.Runtime.loadLibrary(Runtime.java:365)
	at java.lang.System.loadLibrary(System.java:514)
	at com.mapzen.tangram.MapController.<clinit>(MapController.java:961)
	at com.mapzen.tangram.MapView.getMapInstance(MapView.java:107)
	at com.mapzen.tangram.MapView.getMapAsync(MapView.java:77)
	at com.mapzen.tangram.MapView.getMapAsync(MapView.java:61)
	at de.westnordost.streetcomplete.tangram.MapFragment.getMapAsync(MapFragment.java:74)
	at de.westnordost.streetcomplete.tangram.MapFragment.getMapAsync(MapFragment.java:69)
	at de.westnordost.streetcomplete.MainActivity.onCreate(MainActivity.java:185)
	at android.app.Activity.performCreate(Activity.java:5166)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1081)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2309)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2397)
	at android.app.ActivityThread.access$600(ActivityThread.java:156)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1340)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:153)
	at android.app.ActivityThread.main(ActivityThread.java:5338)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:511)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by tallytalwar** [comment]
This is already fixed in master. Refer https://github.com/tangrams/tangram-es/pull/1389 and the reported issue: https://github.com/tangrams/tangram-es/issues/1384.

The fix will be available in our next release 0.5.2 next week.

Thanks, closing this.

**Comment by westnordost** [comment]
Nice
